index.js

```diff
@@ -30,9 +30,22 @@ module.exports.PLAYLIST_ARGUMENTS = ['--flat-playlist', '-j'];
 
 const timescale = 48000;
 
+const generateDurationString = (totalSeconds) => {
+  const secondsString = `${totalSeconds % 60}S`;
+
+  const totalMinutes = Math.floor(totalSeconds / 60);
+  const minutes = totalMinutes % 60;
+  const minutesString = minutes > 0 ? `${minutes}M` : '';
```

Comment:
is skipping minutes like this valid if hours are present? e.g. `PT1H30S`

Reply:
can't be bothered pirating the ISO spec to find out, guess we'll find out what gstreamer does

Reply:
I've seen manifests that don't have seconds so I assume it's valid.